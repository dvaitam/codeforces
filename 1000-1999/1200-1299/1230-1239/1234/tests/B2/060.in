29 4
13 6 13 8 4 8 11 11 8 15 16 12 16 7 14 15 13 18 4 19 16 9 5 5 1 13 14 4 1
