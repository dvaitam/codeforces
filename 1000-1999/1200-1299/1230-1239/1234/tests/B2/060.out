4
4 14 13 1
